app/javascript/dashboard/routes/dashboard/settings/inbox/PreChatForm/PreChatFields.vue

```diff
@@ -45,9 +46,9 @@ export default {
       <tr class="border-b border-n-weak">
         <td class="pre-chat-field"><fluent-icon icon="drag" /></td>
         <td class="pre-chat-field">
-          <woot-switch
-            :model-value="item['enabled']"
-            @input="handlePreChatFieldOptions($event, 'enabled', item)"
+          <ToggleSwitch
+            v-model="item['enabled']"
```

Comment:
Instead of binding the v-model, let use `:model-value`

```suggestion
            :model-value="item['enabled']"
```

The `defineModel` inside the `Switch.vue` will automatically declare a prop with the same name and a corresponding update:propName event.